kubernetes/charts/edge-kubernetes/values.yaml

```diff
@@ -163,7 +163,7 @@ edgeAgent:
   containerName: edgeagent
   image:
     repository: azureiotedge/azureiotedge-agent
-    tag: 0.1.0-beta13
+    tag: 0.1.0-beta14ß
```

Comment:
Is the `ß` at the end intentional?

Reply:
I don't think so. rouge character :)

Reply:
It's called hitting option-S instead of command-S on a Mac....